03-Binary-Search-Trees/02-Binary-Search-Trees.md

```diff
@@ -23,7 +23,7 @@ The `ListNode` class was used in a larger `LinkedList` class which maintained a
 
 Binary search trees have a _non-linear structure_. In a binary search tree each node has pointers to two other nodes in the data structure. We refer to these pointers as the _child_ nodes of the original _parent_ node. We label each of the parent node's two children as the `left` and `right` children or pointers. Collectively, we can refer to a parent node's children as _siblings_.
 
-A binary search tree's `left` and `right` child nodes must maintain special properties. The `left` child must have a `key` that is less than or equal to the `key` of its parent node. The `right` child must have a `key`  that is greater than or equal to that of its parent node.
+A binary search tree's `left` and `right` child nodes must maintain special properties. The `left` child must have a `key` that is less than the `key` of its parent node. The `right` child must have a `key`  that is greater than or equal to that of its parent node.
```

Comment:
I'm okay with making this change however both are valid, they should just make a consistent choice across the board. We could consider instead highlighting that either is valid as long as the choice is consistent rather than just making it 'greater than or equal to' across the board

Reply:
Good callout! I updated the verbiage to reflect this 🙌🏾